datanode/src/main/java/com/odysseusinc/arachne/system/settings/api/v1/controller/SystemSettingsController.java

```diff
@@ -15,24 +15,22 @@
 
 package com.odysseusinc.arachne.system.settings.api.v1.controller;
 
-import com.odysseusinc.arachne.commons.api.v1.dto.util.JsonResult;
-import com.odysseusinc.arachne.system.settings.api.v1.dto.SystemSettingsGroupDTO;
 import com.odysseusinc.arachne.system.settings.api.v1.dto.SystemSettingsGroupListDTO;
 import com.odysseusinc.arachne.system.settings.api.v1.dto.SystemSettingsListDTO;
 import com.odysseusinc.arachne.system.settings.exception.NoSuchSystemSettingException;
 import com.odysseusinc.arachne.system.settings.model.SystemSettingsGroup;
 import com.odysseusinc.arachne.system.settings.service.SystemSettingsService;
 import io.swagger.annotations.Api;
-import java.util.LinkedList;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.core.convert.support.GenericConversionService;
 import org.springframework.security.access.annotation.Secured;
 import org.springframework.validation.BindException;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.LinkedList;
```

Comment:
Minor: Why did you choose LinkedList instead of ArrayList?

Reply:
I didn't, it's just import reordering